congress/metadata/K000390.yaml

```diff
@@ -0,0 +1,2 @@
+name: Votesmart
+link: http://votesmartnv.org/us-congress/ruben-kihuen/
```

Comment:
This is from the campaign. Candidates for replacement:

* Home page header: https://kihuen.house.gov/images/headshot-web.png
* Twitter avatar: https://twitter.com/RepKihuen